src/js/components/site-alert.js

```diff
@@ -0,0 +1,20 @@
+const behavior = require("../utils/behavior");
+const { CLICK } = require("../events");
+
+const { prefix: PREFIX } = require("../config");
+
+const SITE_ALERT_CLASS = `${PREFIX}-site-alert`;
+
+const SITE_ALERT = `.${SITE_ALERT_CLASS}`;
+const CLOSE_BUTTON = `.${SITE_ALERT_CLASS}__close-btn`;
+
+const closeBanner = function toggleEl(event) {
+    event.preventDefault();
+    this.closest(SITE_ALERT).hidden = true;
```

Comment:
Is there any good way to begin to build a persistent close-state into this component? Is there a good way to use cookies, data attributes, or something to make a persistent close functional by default?

Reply:
Local storage could be an option as well. They would also need to give it some unique identifier so that all future banners wouldn't be hidden. https://medium.com/datadriveninvestor/cookies-vs-local-storage-2f3732c7d977

Reply:
Would we expect users to know to clear local storage/cookies if they need to see the alert again?

Reply:
Yup. So that's why we would want to make sure the identifier is unique in the case where this was being implemented.